Mammoth/Include/TSEAccessibilitySettings.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include "DirectXUtil.h"
+#include "Kernel.h"
+
+using namespace Kernel;
+
+class CAccessibilitySettings {
+public:
+	enum IFFType {
+		player,
+		escort,
+		friendly,
+		neutral,
+		enemy,
+		angry,
+		projectile,
+	};
+	CAccessibilitySettings (void);
+	~CAccessibilitySettings (void);
+	void SetIFFColor (CString sSettingsValue, int iIFFType);
+	CG32bitPixel GetIFFColor (int iIFFType);
+protected:
+
+private:
+	CG32bitPixel m_rgbColorPlayer;
```

Comment:
I think this is OK, but have you considered an array of C32bitPixels indexed by IFFType? That might make things easier.

Reply:
Upcoming commit implements this as well, tidies up a lot of the function calls with a couple casts.

Reply:
done